workers/unity/ProjectSettings/QualitySettings.asset

```diff
@@ -123,7 +123,7 @@ QualitySettings:
     shadowCascade2Split: 0.33333334
     shadowCascade4Split: {x: 0.06666667, y: 0.2, z: 0.46666667}
     shadowmaskMode: 0
-    blendWeights: 1
+    skinWeights: 1
```

Comment:
why did the name change?

Reply:
Probably re-exported when I build out a client. Upgrade to 2019.1 still had some name changes left.